### PR TITLE
Add individual storage workbook template

### PR DESCRIPTION
**Pull request by acearun**

### CODE REVIEW

**Comment by acearun** [comment]
image

File: Workbooks/Individual Storage/Overview/Overview.workbook [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+          {
+            "resourceId": "{StorageAccount}"
+          }
+        ],
+        "title": "Success end-to-end latency",
+        "gridFormatting": []
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "metric - 0 - Copy - Copy"
```
Comment (vutran01) [inline Workbooks/Individual Storage/Overview/Overview.workbook]: add name for this guy "metric - 0 - Copy - Copy"?

File: Workbooks/Individual Storage/Overview/Overview.workbook [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+          {
+            "resourceId": "{StorageAccount}"
+          }
+        ],
+        "title": "Success server latency",
+        "gridFormatting": []
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "metric - 0 - Copy - Copy - Copy"
```
Comment (vutran01) [inline Workbooks/Individual Storage/Overview/Overview.workbook]: same here?

File: Workbooks/Individual Storage/Overview/Overview.workbook [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+      "name": "text - 13"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "*Success Server Latency* - the average time used to process a successful request by Azure Storage. This value does not include the network latency specified in SuccessE2ELatency. \r\n<br />\r\n<br />"
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "text - 13 - Copy"
```
Comment (vutran01) [inline Workbooks/Individual Storage/Overview/Overview.workbook]: name?

File: Workbooks/Individual Storage/Overview/Overview.workbook [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+      "name": "metric - 0 - Copy - Copy - Copy"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "*Success E2E Latency* - the average end-to-end latency of successful requests made to a storage service or the specified API operation. This value includes the required processing time within Azure Storage to read the request, send the response, and receive acknowledgment of the response. \r\n<br />\r\n<br />"
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "text - 13"
```
Comment (vutran01) [inline Workbooks/Individual Storage/Overview/Overview.workbook]: significant of "13"?

File: Workbooks/Individual Storage/Overview/Overview.workbook [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+          {
+            "resourceId": "{StorageAccount}"
+          }
+        ],
+        "title": "Ingress",
+        "gridFormatting": []
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "metric - 0 - Copy - Copy - Copy - Copy"
```
Comment (vutran01) [inline Workbooks/Individual Storage/Overview/Overview.workbook]: same here?

File: Workbooks/Individual Storage/Overview/Overview.workbook [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+          {
+            "resourceId": "{StorageAccount}"
+          }
+        ],
+        "title": "Egress",
+        "gridFormatting": []
+      },
+      "conditionalVisibility": null,
+      "customWidth": "50",
+      "name": "metric - 0 - Copy - Copy - Copy - Copy - Copy"
```
Comment (vutran01) [inline Workbooks/Individual Storage/Overview/Overview.workbook]: same here

File: Workbooks/Individual Storage/Overview/Overview.workbook (hunk 1 above)
Comment (acearun) [reply Workbooks/Individual Storage/Overview/Overview.workbook]: Fixed

File: Workbooks/Individual Storage/Overview/Overview.workbook (hunk 2 above)
Comment (acearun) [reply Workbooks/Individual Storage/Overview/Overview.workbook]: Fixed

File: Workbooks/Individual Storage/Overview/Overview.workbook (hunk 3 above)
Comment (acearun) [reply Workbooks/Individual Storage/Overview/Overview.workbook]: Fixed

File: Workbooks/Individual Storage/Overview/Overview.workbook (hunk 4 above)
Comment (acearun) [reply Workbooks/Individual Storage/Overview/Overview.workbook]: Fixed

File: Workbooks/Individual Storage/Overview/Overview.workbook (hunk 5 above)
Comment (acearun) [reply Workbooks/Individual Storage/Overview/Overview.workbook]: Fixed

**Pull request merged**